pkg/asset/templates/kubelet.yaml

```diff
@@ -17,22 +20,9 @@ spec:
       - name: kubelet
         image: quay.io/coreos/hyperkube:v1.3.6_coreos.0
         command:
-        - /nsenter
-        - --target=1
-        - --mount
-        - --wd=.
-        - --
-        - ./hyperkube
-        - kubelet
-        - --api-servers={{ index .APIServers 0 }}
-        - --config=/etc/kubernetes/manifests
-        - --allow-privileged
-        - --hostname-override=$(MY_POD_IP)
-        - --cluster-dns=10.3.0.10
-        - --cluster-domain=cluster.local
-        - --kubeconfig=/etc/kubernetes/kubeconfig
-        - --lock-file=/var/run/lock/kubelet.lock
-        - --minimum-container-ttl-duration=3m0s
+        - /bin/bash
+        - -c
+        - "source /etc/kubernetes/kubelet.env; /nsenter --target=1 --mount --wd=. -- ./hyperkube kubelet --hostname-override=$(MY_POD_IP) $KUBELET_OPTS"
```

Comment:
Let's switch this back now that we're going to use upstream work for updating kubelet config